README.md

```diff
@@ -21,6 +21,8 @@ To get a local copy up and running follow these simple example steps.
 
 - Ruby
 
+### project presentation [Video](https://drive.google.com/file/d/1_CRYO020mGg-_-eJVoyG2rWjquYgKl4X/view?usp=sharing)
+

```

Comment:
- [ ] I see that in the video this is a two members group, but I don't see any commit from the other partner or even as a contributor for this repository, please make sure that both member worked on the project.

<img width="680" alt="Screen Shot 2022-07-30 at 1 53 21 PM" src="https://user-images.githubusercontent.com/83717285/181937785-349645a7-1f42-4b85-a239-10bb73f57e20.png">